src/test/smart-components/group/remove-group-modal.test.js

```diff
@@ -11,24 +11,26 @@ import { RBAC_API_BASE } from '../../../utilities/constants';
 import RemoveGroupModal from '../../../smart-components/group/remove-group-modal';
 import { groupsInitialState } from '../../../redux/reducers/group-reducer';
 import { REMOVE_GROUP, FETCH_GROUP } from '../../../redux/action-types';
+import { Button, Checkbox } from '@patternfly/react-core';
 
 describe('<RemoveGroupModal />', () => {
   let initialProps;
   const middlewares = [ thunk, promiseMiddleware(), notificationsMiddleware() ];
   let mockStore;
   let initialState;
 
-  const GroupWrapper = ({ store, children }) => (
+  const GroupWrapper = ({ store }) => (
     <Provider store={ store }>
       <MemoryRouter initialEntries={ [ '/groups/', '/groups/123/', '/groups/' ] } initialIndex={ 1 }>
-        { children }
+        <Route path="/groups/removegroups" render={ (args) => <RemoveGroupModal { ...args } { ...initialProps }
+          isModalOpen
+          groupsUuid={ [{ uuid: '1234' }] } /> } />
```

Comment:
This should be the same as what you are expecting in mock API. So `123` based on the rest of the code around.

Reply:
Good catch! Totally missed it!